dt-core/global-functions.php

```diff
@@ -694,10 +695,7 @@ function render_field_for_display( $field_key, $fields, $post, $show_extra_contr
                     </dt-single-select>
             <?php elseif ( $field_type === 'tags' ): ?>
                 <?php $value = array_map(function ( $value ) {
-                    return [
-                        'id' => $value,
-                        'label' => $value,
-                    ];
+                    return $value;
```

Comment:
@micahmills,

 This code is related to the dt-tag field on the edit detail page. We made this change to make the DT-tag field visible on frontend(UI) also user is able to edit the same. 
Related code in web components is in this PR also
https://github.com/DiscipleTools/disciple-tools-web-components/pull/76#issuecomment-2182218619

We have made changes on both sides (DT-Theme as well as Web components)  are related to this dt-tags fix. 

Kindly check and let us know your feedback. Thank you!

cc: @gp-birender @vijenderrr

Reply:
Thank you @Ashima-Arora it was my mistake I thought I was testing the Web component PR with this branch, but I was not. I will merge this in and we can deal with the location_meta stuff at a later time.